compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/test/AMDAIEDmaUtilsTest.cpp

```diff
@@ -241,6 +255,12 @@ TEST_F(AccessPatternCombinationTest, CombineAccessPatterns) {
   checkCombineAccessPatterns({2, 32}, {16, 32}, {16, 1}, {0, 6, 32},
                              {2, 16, 32}, {64, 16, 1}, {0, 2, 32}, {3, 16, 32},
                              {64, 16, 1}, 4);
+  checkCombineAccessPatterns({0}, {32}, {1}, {1, 0}, {2, 32}, {64, 1}, {0, 0},
+                             {3, 32}, {64, 1}, 3);
+  // size(B) > size(A) Same access pattern
+  checkCombineAccessPatterns({0, 0}, {16, 32}, {16, 1}, {1, 0, 0}, {3, 16, 32},
```

Comment:
If B has offset {2, 0, 0}, what is the expected offset of the combined output? Or does the merge fail?

Reply:
It will fail, I can add that case as well.

Reply:
Thanks